core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1993,4 +1993,15 @@ private SqlLibraryOperators() {
                   SqlTypeFamily.INTEGER),
               number -> number == 2),
           SqlFunctionCategory.NUMERIC);
+
+  @LibraryOperator(libraries = {SNOWFLAKE})
+  public static final SqlFunction SPLIT_WITH_INDEX = new SqlFunction(
```

Comment:
Need to add test case for the same in Calcite code also.

Reply:
As discussed with Anurag, this is a temporary solution till GUDU issue is fixed. This code will be removed later. Hence allowing without test case.